### DISCUSSION TITLE
The value for setDefaultOptions is lost after refreshing

### DISCUSSION THREAD

**Discussion by alexandrapopa96** [js-fns/date-fns#3409] [General]
Hi, 

I am using setDefaultOptions() function to set the default locale to the current language of a user after logging in and works perfect, but after refreshing, the locale is reseted to the initial default one, in english. 
Can't this value be persistent? Or is there any other way to do the setting described as needed?

Thanks!

**Comment by marchiartur** [comment]
> Hi,
> 
> I am using setDefaultOptions() function to set the default locale to the current language of a user after logging in and works perfect, but after refreshing, the locale is reseted to the initial default one, in english. Can't this value be persistent? Or is there any other way to do the setting described as needed?
> 
> Thanks!

you can save this information in browser cache

**Comment by fturmel** [comment]
Persist the user choice in localStorage or a cookie, or as part of their user preferences if you authenticate them and have a profile in some database.
